src/sparsezoo/analyze/analysis.py

```diff
@@ -567,6 +592,197 @@ def _sum_across_keys(dict, key):
         )
 
 
+class CountSummary(BaseModel):
+    items: List[_SparseItemCount]
+    _precision: int = 2
+
+    @property
+    def total(self):
+        return sum(item.total for item in self.items)
+
+    @property
+    def sparse_count(self):
+        return sum(item.pruned for item in self.items)
+
+    @property
+    def dense_count(self):
+        return sum(item.dense for item in self.items)
+
+    @property
+    def quant_count(self):
+        return sum(item.quantized for item in self.items)
+
+    @property
+    def sparsity(self):
+        return round(self.sparse_count * 100.0 / self.total, self._precision)
+
+    @property
+    def quantized(self):
+        return round(self.quant_count * 100.0 / self.total, self._precision)
+
+    @property
+    def size(self):
+        """
+        :return: size in bits ignoring zeros
+        """
+        return self.quant_count * 8 + self.dense_count * 32
```

Comment:
First of all, I do not like using the name dense for something not quantised. Those two terms are not antonyms.
Secondly, is it safe to hard code the mapping quant-> 8bit, non-quant(full precision)->32bit? This could be very likely also 16bit no?

Reply:
For now we decided to keep it such, but yes I agree, this will change in the future to include other precisions too